packages/deno/src/utils/assets/copy-assets-handler.ts

```diff
@@ -0,0 +1,203 @@
+import * as minimatch from 'minimatch';
+import * as path from 'path';
+import * as fse from 'fs-extra';
+import ignore from 'ignore';
+import * as fg from 'fast-glob';
```

Comment:
if we end up keeping a copy of these asset utils in the deno plugin, then we'll need to make sure all of these deps are included in the deno plugins package.json, but I'd prefer to just include @nx/js as a dep and use it's asset handlers